code/modules/atmospherics/gasmixtures/gas_types_hardcoded.dm

```diff
@@ -195,7 +195,7 @@
 	id = GAS_ID_VIMUR
 	name = "Vimur"
 	specific_heat = 500	// J/(mol*K) //250% the heat capacity of phoron
-	molar_mass = 0.054 // Standard Mass of xenon
+	molar_mass = 1000 * 0.054 // Standard Mass of xenon
```

Comment:
got lazy with the conversion huh?

Reply:
the entire hardcoded file is basically a "we shoudl review and seriously re-evaluate these later since these were justported"; the types/whatever files are meant to be in house and basically finalized in stats

Reply:
okay